source/docs/software/vision-processing/using-multiple-cameras.rst

```diff
@@ -94,13 +88,6 @@ streams open, even if you aren't using them.
           prevTrigger = joy1.GetTrigger();
         }
 
-If both cameras are USB, it's worth noting that you may run into USB bandwidth limitations with higher resolutions, as in
-all of these cases the roboRio is going to be streaming data from both cameras to the roboRio simultaneously (for a short
-period in options 1 and 2, and continuously in option 3). It is theoretically possible for the library to avoid this
-simultaneity in the option 2 case (only), but this is not currently implemented.
+If both cameras are USB, it's worth noting that you may run into USB bandwidth limitations with higher resolutions, as in all of these cases the roboRIO is going to be streaming data from both cameras to the roboRIO simultaneously (for a short period in options 1 and 2, and continuously in option 3). It is theoretically possible for the library to avoid this simultaneity in the option 2 case (only), but this is not currently implemented.
 
-Different cameras report bandwidth usage differently. The library will tell you if you're hitting the limit; you'll get
-this error message: "could not start streaming due to USB bandwidth limitations; try a lower resolution or a different
-pixel format (VIDIOC_STREAMON: No space left on device)". If you're using Option 3 it will give you this error during
-RobotInit(). Thus you should just try your desired resolution and adjusting as necessary until you both don't get that
-error and don't exceed the radio bandwidth limitations.
+Different cameras report bandwidth usage differently. The library will tell you if you're hitting the limit; you'll get this error message: "could not start streaming due to USB bandwidth limitations; try a lower resolution or a different pixel format (VIDIOC_STREAMON: No space left on device)". If you're using Option 3 it will give you this error during RobotInit(). Thus you should just try your desired resolution and adjusting as necessary until you both don't get that error and don't exceed the radio bandwidth limitations.
```

Comment:
Would it be a good idea to surround areas such as `VIDIOC_STREAMON: No space left on device` with 
```
`VIDIOC_STREAMON: No space left on device`
```
And also `RobotInit()` to place emphasis?

I'm open to different ideas.

Reply:
Yes, I think we should always do literals for function names.  The message is more than just the STREAMON part, it’s everything in quotes.  I think maybe a better approach for that is a block literal instead of inline, since it’s so long.